gobuffalo/azuredeploy.json

```diff
@@ -2,9 +2,16 @@
     "$schema": "https://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
+      "location" : {
+        "type": "String",
+        "defaultValue": "[resourceGroup().location]",
+        "metadata": {
+          "description": "The Azure Region which should be targeted while provisioning the infrastructure described in this template."
+        }
+      },
       "imageName" : {
         "type": "String",
-        "defaultValue": "appsvc/sample-hello-world:latest",
+        "defaultValue": "marstr/quickstart:latest",

```

Comment:
need a "community" image here, not a personal one...

Reply:
Fixed locally, I'll update this back to the quickstart once it has a better landing place.